keps/sig-node/2400-node-swap/README.md

```diff
@@ -1188,6 +1336,8 @@ nodes that do not use swap memory.
 - **2023-04-17:** KEP update for beta1 [#3957](https://github.com/kubernetes/enhancements/pull/3957).
 - **2023-08-15:** Beta1 released in kubernetes 1.28
 - **2024-01-12:** Updates to Beta2 KEP.
+- **2024-01-08:** Beta2 released in kubernetes 1.30.
```

Comment:
tmpfs noswap support went in for 1.31 right?

Reply:
Yes, that's correct (was done in https://github.com/kubernetes/kubernetes/pull/124060)

Reply:
So should we mention that in the timestamps?

Reply:
We technically did some work on it for 1.31 that was critical for this feature.

Reply:
I thought this section just outlines how the KEP itself advanced and not every implementation that have happened. But yeah, that makes total sense.

Let me add more data to this section. I'll also add data about previous implementations as well :+1:

Reply:
Thanks again for catching that @kannon92!
Done :) PTAL